### ISSUE TITLE
Design System Website - Get Started

### ISSUE THREAD

**Issue by ArakTaiRoth** [GovAlta/ui-components#1232]
## Acceptance Criteria:

- The all pages included in the Figma design will be created under the top level navigation item "Get Started":
  - QA testing setup and Open forum are out of scope
  - The following pages under "UX Designers" are also out of scope
    - Setup
    - Figma libraries
    - Fonts
    - UX writing
- Breadcrumbs are not included in the MVP

[Figma Design](https://www.figma.com/file/qWGWDvULAAZuVaDpXCRzqP/DS-Resource-Website?type=design&node-id=3605%3A804701&mode=design&t=DkDfZnRPTQj6IVjA-1)

**Comment by ArakTaiRoth** [comment]
In blocked state until #1230 is complete. Once that is complete, this can be moved to Review, and [PR](https://github.com/GovAlta/ui-components-docs/pull/10) can be marked as ready for review.
